src/v2/view-builder/internals/BaseFooter.js

```diff
@@ -62,9 +63,11 @@ export default View.extend({
     }
 
     links.forEach(link => {
-      this.add(Link, {
-        options: link,
-      });
+      let LinkView = Link;
+      if (link.type === 'toggle-text-link') {
+        LinkView = ToggleTextLink;
+      }
+      this.add(LinkView, { options: link, });
```

Comment:
nit: remove the extra comma at the end, or leave it as it was before in a different line